site/en/docs/extensions/whatsnew/index.md

```diff
@@ -16,6 +16,27 @@ updated: 2021-11-24
 Check this page often to learn about changes to the Chrome extensions platform, its documentation,
 and related policy or other changes.
 
+### 2022.01.05: `match_origin_as_fallback` in Canary {: #canary-match-origin-as-fallback }
+
+Content scripts can now specify the `match_origin_as_fallback` key to inject into frames that are
+related to a matching frame, including frames with `about:`, `data:`, `blob:`, and `filesystem:`
+URLs.  See the [content scripts](/docs/extensions/mv3/content_scripts/#injecting-in-related-frames)
+documentation for details.
+
+### 2021.12.30:  in Canary {: #canary-file-access }
+
+Service worker based Manifest V2 and Manifest V3 extensions can now use the Fetch API to request
+`file://` URLs. Access to `file://` URLs still requires that the user enable 'Allow access to File
```

Comment:
```suggestion
`file:`-scheme URLs. Access to `file:`-scheme URLs still requires that the user enable 'Allow access to File
```

I think "`file:`-scheme URLs" might be more accurate.  The number of slashes after it is a subtle topic.